cupy/cuda/cupy_thrust.cu

```diff
@@ -126,31 +126,47 @@ __host__ __device__ __forceinline__ bool _cmp_less(const T& lhs, const T& rhs) {
 
 // specialize thrust::less for single complex
 template <>
-__host__ __device__ __forceinline__ bool less<complex<float>>::operator() (
+__host__ __device__ __forceinline__
+#if (__CUDACC_VER_MAJOR__ == 11 && __CUDACC_VER_MINOR__ == 2) 
```

Comment:
This should've been 
```C++
#if (__CUDACC_VER_MAJOR__ == 11 && __CUDACC_VER_MINOR__ >= 2)
```
I can send a PR after code freeze.

Reply:
Better:
```
#if (__CUDACC_VER_MAJOR__ >11 || (__CUDACC_VER_MAJOR__ == 11 && __CUDACC_VER_MINOR__ >= 2))
```

Reply:
Changed, thanks!